README.md

```diff
@@ -8,9 +8,16 @@
 
 Flake8 Numbers Plugin
 
-This repository is currently in bootstrapping phase, expect it to be flakey 😊.
+This plugin ensures that large numbers of all bases are using digit seperators for improved readability.
 
-TODO: Improve the README file
+Some examples of well written numeric literals are:
+
+```python
+good_float   = 123_456_789.123_456_789
```

Comment:
I'm now wondering what's the correct format for `1234.1234`, I guess it is `1_234.123_4`, right?